### DISCUSSION TITLE
Carousel for gallery thumbnails on PDP

### DISCUSSION THREAD

**Discussion by pk-pankaj** [breezefront/community#59] [1. Help]
Hi @vovayatsyuk 

I have added a carousel for the gallery thumbnails on PDP in the breeze theme and it's working fine initially, I used like below in the gallery.phtml

`<script type="text/x-magento-init">

        ".breeze-gallery .thumbnails": {
            "Magento_PageBuilder/js/content-type/slider/appearance/default/widget":{
                "arrows": true,
                "dots": false
            }
        }
    }
</script>`

But not working for the configurable products when we select options and thumbnails are regenerated.

Please help with this if possible.

Thanks

**Comment by vovayatsyuk** [comment]
I recommend sticking to the default implementation (without slider).

If you are trying to add arrows, you could add them from custom.js with custom click listeners that will call native scrollTo or scrollBy method.

**Reply by pk-pankaj** [reply]
ok, I will try.

Thanks @vovayatsyuk
